Description: Fix undefined use of sprintf
 This patch fixes the undefined use of sprintf when the
 source and destination buffers overlap.
Author: Graham Inggs <graham@nerve.org.za>
Forwarded: http://bugs.motifzone.net/show_bug.cgi?id=1628
Last-Update: 2013-09-24
--- a/lib/Xm/XmRenderT.c
+++ b/lib/Xm/XmRenderT.c
@@ -2703,7 +2703,7 @@
       number = tlist -> count;
       tab = (_XmTab) tlist -> start;
       while(number > 0) {
-	sprintf(temp, "%s %f %d %d %d, ", temp, tab -> value, 
+	sprintf(temp + strlen(temp), " %f %d %d %d, ", tab -> value,
 		tab -> units, tab -> alignment, tab -> offsetModel);
 	tab = (_XmTab) tab -> next;
 	number--;
